src/applications/hca/containers/IntroductionPage.jsx

```diff
@@ -1,156 +1,176 @@
 
 import { focusElement } from 'platform/utilities/ui';
+import { toggleLoginModal } from 'platform/site-wide/user-nav/actions';
+import recordEvent from 'platform/monitoring/record-event';
 import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+import environment from 'platform/utilities/environment';
+
+import HCAEnrollmentStatus from './HCAEnrollmentStatus';
+import HCASubwayMap from '../components/HCASubwayMap';
+import {
+  isLoading,
+  isLoggedOut,
+  isUserLOA1,
+  isUserLOA3,
+  shouldShowLoggedOutContent,
+} from '../selectors';
+
+const VerificationRequiredAlert = () => (
+  <AlertBox
+    content={
+      <div>
+        <h4 className="usa-alert-heading">
+          Please verify your identity before applying for VA health care
+        </h4>
+        <p>This process should take about 5 to 10 minutes.</p>
+        <p>
+          <strong>If you’re applying for the first time</strong>
+        </p>
+        <p>
+          We need to verify your identity so we can help you track the status of
+          your application once you’ve submitted it. As soon as you’re finished
+          verifying your identity, you can continue to the application.
+        </p>
+        <p>
+          <strong>If you’ve applied before</strong>
+        </p>
+        <p>
+          We need to verify your identity so we can show you the status of your
+          past application. We take your privacy seriously, and we need to make
+          sure we’re sharing your personal information only with you.
+        </p>
+        <p>
+          <strong>
+            If you need more information or help with verifying your identity:
+          </strong>
+        </p>
+        <ul>
+          <li>
+            <a href="/sign-in-faq/#verifying-your-identity">
+              Read our identity verification FAQs
+            </a>
+          </li>
+          <li>
+            Or call us at 877-222-VETS (
+            <a href="tel:+18772228387">877-222-8387</a>
+            ). If you have hearing loss, call TTY: 800-877-8339. We’re here
+            Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.
+          </li>
+        </ul>
+        <p>
+          <a
+            className="usa-button-primary va-button-primary"
+            href="/verify"
+            onClick={() => {
+              recordEvent({ event: 'verify-link-clicked' });
+            }}
+          >
+            Verify your identity
+          </a>
+        </p>
+      </div>
+    }
+    isVisible
+    status="continue"
+  />
+);
+
+const LoggedOutContent = connect(
+  null,
+  { toggleLoginModal },
+)(({ route, showLoginAlert, toggleLoginModal: showLoginModal }) => (
+  <>
+    {showLoginAlert && (
+      <div>
+        <AlertBox
+          headline="Have you applied for VA health care before?"
+          content={
+            <button
+              className="va-button-link"
+              onClick={() => showLoginModal(true, 'hcainfo')}
```

Comment:
Could push some of this into `mapDispatchToProps` e.g. 

`{ toggleLoginModal }` could be `{ loginAlertClicked: () => toggleLoginModal(true, 'hcainfo') }`
then I think the component only needs to pass the callback directly to `onClick` (no need to couple it to `toggleLoginModal`):

```
)(({ route, showLoginAlert, showLoginAlertClicked }) => (
...
<button
  ...
  onClick={showLoginAlertClicked}
```